Nim/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Welcome to Nim! Stick draw game, 1 to 3 sticks to draw");
+Random random = new Random();
+int matches = 24;
+// string match = "|";''
+
+goback:
+
+Console.WriteLine($"there are this many matches: " + new string('|', matches) + $" ({matches})");
+if (matches > 1)
+{
+    Console.WriteLine("How many matches do you draw?");
+}
+
+
+switch (matches)
+{
+    case > 1:
+
+        retry:
+        int take;
+        bool success = int.TryParse(Console.ReadLine(), out take);
+        if (!success)
+        {
+            Console.WriteLine("Enter a number between 1 and 3 please!");
+            goto retry;
+        }
+
+        switch (take)
+        {
+            case > 3:
+            case < 1:
+                Console.WriteLine("Enter a number between 1 and 3 please!");
+                goto goback;
+
+        }
+
+        reroll:
+        int bot = random.Next(1, 4);
+        if (bot > matches)
+        {
+            goto reroll;
+        }
+        
```

Comment:
This can be done smarter. Just make sure the bot doesn't roll higher than `matches` in the first place ;)